packages/wonder-blocks-core/util/util.js

```diff
@@ -60,3 +64,33 @@ export function processStyleList<T: Object>(
         className: css(...stylesheetStyles),
     };
 }
+
+export const mediaContextTypes = {
+    mediaSize: propTypes.string,
+    mediaSpec: propTypes.object,
+};
+
+export function MediaLayoutWrapper<Props: {}>(
```

Comment:
Maybe add a TODO to change this to use the FAC pattern.

Reply:
Hmm - I could, although it's probably just going to be my next PR (and I'm not expecting anyone to use it in the meantime).